lib/tasks/data_migrations/migrate_contact_metadata_to_native_fields.rb

```diff
@@ -0,0 +1,42 @@
+namespace :data_migrations do
+  task migrate_contact_metadata_to_native_fields: :environment do
+    Account.find_each do |account|
+      puts "Migrating contacts for account: #{account.id}"
+
+      account.contacts.find_each do |contact|
+        ApplicationRecord.transaction do
+          contact.iso_country_code = PhonyRails.country_from_number(contact.msisdn)
+          contact.language_code = contact.metadata["language_code"]
+          contact.save!
+
+          # EWS Cambodia
+          contact.metadata.fetch("commune_ids", []).each do | commune_id |
+            commune = Pumi::Commune.find_by_id(phone_call_metadata(:commune_code))
+
+            contact.addresses.find_or_create_by!(
+              iso_region_code: commune.province.iso3166_2,
+              administrative_division_level_2_code: commune.district_id,
+              administrative_division_level_3_code: commune.id
+            )
+          end
+
+          # EWS Laos
+          contact.metadata.fetch("registered_districts", []).each do | district_code |
+            district = CallFlowLogic::EWSLaosRegistration::DISTRICTS.find { |d| d.code ==  district_code }
+
+            contact.addresses.find_or_create_by!(
+              iso_region_code: district.province.iso3166,
```

Comment:
`district.province.iso3166_2`

Reply:
For Laos, there is no field/method with `iso316_2`.
![CleanShot 2024-12-19 at 08 50 57@2x](https://github.com/user-attachments/assets/d86fe94d-b5c6-423c-8d27-91b26fcc0678)